packages/cspell/src/cli-reporter.ts

```diff
@@ -0,0 +1,166 @@
+import chalk = require('chalk');
```

Comment:
cli-reporter is based on  current `getEmitters()` implementation from master
I did not export it as a separate package because it's strongly tied to cli configuration